cit3.0-web/src/components/Page/ReviewOpportunity/ReviewOpportunity.stories.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
```

Comment:
Same as earlier, why disable for this file?

Reply:
Removed. Was used before Provider wrapper was added.